.circleci/config.yml

```diff
@@ -0,0 +1,66 @@
+version: 2
+jobs:
+  build:
+    machine: true
+
+    working_directory: ~/data.world-py
+
+    environment:
+      PRERELEASE_BRANCH: prerelease
+      RELEASE_BRANCH: release
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - main_cache
+            - pyenv-{{ checksum "tox.ini" }}
+
+      - run:
+          name: Install pyenv versions
+          command: |
+            pyenv versions
+            pyenv install --list
+            pyenv install --skip-existing 2.7.13
+            pyenv install --skip-existing 3.4.5
+            pyenv install --skip-existing 3.5.2
+            pyenv install --skip-existing 3.6.0
+
+      - save_cache:
+          key: pyenv-{{ checksum "tox.ini" }}
+          paths: /opt/circleci/.pyenv
+
+      - run:
+          name: Install python dependencies
+          command: |
+            pyenv global 2.7.13 3.4.5 3.5.2 3.6.0
+            pip install tox twine
+
+      - run: env
+
+      - run:
+          name: tox
+          command: tox
+
+      - save_cache:
+          key: main_cache
+          paths:
+            - .eggs
+            - .tox
+
+      - deploy:
+          name: Pre-release to pypi
+          command: |
+            if [[ "${CIRCLE_BRANCH}" =~ ^(${PRERELEASE_BRANCH})$ ]]; then
+              echo 'Do a prerelease with twine here'
```

Comment:
I think the idea is to release from `PRERELEASE_BRANCH` exactly the same way we release from `RELEASE_BRANCH`. Up to you if you want to repeat or merge the two deploy blocks.